gnucash/gnome/gnc-split-reg.c

```diff
@@ -546,6 +547,63 @@ gsr_redraw_all_cb (GnucashRegister *g_reg, gpointer data)
     gsr_update_summary_label( gsr->projectedminimum_label,
                               xaccAccountGetProjectedMinimumBalance,
                               leader, print_info, commodity, reverse, euro );
+
+    // Sort label
+    if (gsr->sort_label != NULL)
+    {
+        GtkStyleContext *context = gtk_widget_get_style_context (GTK_WIDGET(gsr->sort_label));
+        gchar *text = NULL;
+
+        switch (gsr->sort_type)
+        {
+        case (0):
+            text =  _("None");
+            break;
+        case (1):
+            text = _("Standard Order");
+            break;
+        case (2):
+            text = _("Date");
+            break;
+        case (3):
+            text = _("Date of Entry");
+            break;
+        case (4):
+            text = _("Statement Date");
+            break;
+        case (5):
+            text = _("Number");
+            break;
+        case (6):
+            text = _("Amount");
+            break;
+        case (7):
+            text = _("Memo");
+            break;
+        case (8):
+            text = _("Description");
+            break;
+        case (9):
+            text = _("Action");
+            break;
+        case (10):
+            text = _("Notes");
+            break;
+        }
+
+        if (gsr->sort_rev)
+        {
+            gtk_style_context_add_class (context, "summary-highlight");
+            gtk_widget_set_tooltip_text (GTK_WIDGET(gsr->sort_label), _("Descending"));
```

Comment:
It would be even nicer if the Ascending/Descending information was directly visible in the summary bar. I understand the full text would be too long. However perhaps you can add an image of an up-arrow or down-arrow next to the label (right-aligning the label to ensure the text always sticks to the arrow image). We are using arrow images in the register code also. It would be nice to use the same one for consistency.

Reply:
Originally I was looking to use a font character 'arrow' to do the same but could not find a suitable one. Now you think it is a reasonable idea I will have look at adding an arrow.